test/test_daemon_start.py

```diff
@@ -123,18 +124,19 @@ def get_login_and_group(self, p):
             # so bypass it and keep default value
             return
 
-    def create_daemon(self):
+    def create_daemon(self, is_daemon=False, do_replace=False):
```

Comment:
It was to make a test ... the daemon class has this initialization parameter.  Isn't it necessary?